gax/src/main/java/com/google/api/gax/rpc/ClientSettings.java

```diff
@@ -245,6 +250,11 @@ public B setQuotaProjectId(String quotaProjectId) {
       return self();
     }
 
```

Comment:
Please add Javadoc here.

Reply:
Can we use checkstyle to automatically check for javadoc on public methods?

Reply:
Not sure how flexible the checkstyle plugin is, as we would need to exclude a lot of classes marked with `InternalApi`, but we can give it a try.

Reply:
added javadoc